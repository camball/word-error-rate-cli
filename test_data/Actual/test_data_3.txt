Agent: I won't go to the stores.