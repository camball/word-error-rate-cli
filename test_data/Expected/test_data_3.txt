Agent: I will not go to the store.